echo-telemetry/src/main/java/com/netflix/spinnaker/echo/telemetry/TelemetryEventListener.java

```diff
@@ -217,4 +223,71 @@ public String toString() {
       return new String(getBytes(), StandardCharsets.UTF_8);
     }
   }
+
+  // Arbitrary container for TelemetryEventListener structured data, so there aren't huge
+  // fully-qualified names for the equivalent classes in the kork.proto package all over the code.
+  public static class Holder {
+
+    @Getter
+    @Builder
+    @JsonDeserialize(builder = Content.ContentBuilder.class)
+    public static class Content {
+      @Builder.Default private Execution execution = Execution.builder().build();
```

Comment:
Now that these classes are immutable :tada: :tada: , all the member variables can/should be made `final`.